privacyidea/api/system.py

```diff
@@ -382,3 +445,43 @@ def list_ca_connectors():
     ca_list = get_caconnector_list(return_config=False)
     g.audit_object.log({"success": True})
     return send_result(ca_list)
+
+
+@system_blueprint.route("/nodes", methods=['GET'])
```

Comment:
i think this should require authorization or admin only

Reply:
Done in 200d355

Reply:
Done in 658afd5